tests/test_core.py

```diff
@@ -168,6 +169,32 @@ def test_get_user(self, openshift):  # noqa
         l = openshift.get_user()
         assert l.json() is not None
 
+    def test_watch_resource_and_wait_to_build_timeouts(self, caplog, openshift):  # noqa
```

Comment:
Is this `# noqa:F811`? Although a lot of places say '# noqa', it would be nicer to explicitly silent the particular diagnostic we know about.